File: competitive_programming/codeforces/06_javascript/112a_petya_and_strings.js
function petyaAndStrings(a, b) {
  const left = a.toLowerCase();
  const right = b.toLowerCase();
  if (left < right) return -1;
  if (left > right) return 1;
  return 0;
}

if (require.main === module) {
  if (petyaAndStrings("aaaa", "aaaA") !== 0) throw new Error("petya failed");
  if (petyaAndStrings("abs", "Abz") !== -1) throw new Error("petya failed");
  if (petyaAndStrings("abcdefg", "AbCdEfF") !== 1) throw new Error("petya failed");
  console.log("112A petya and strings ok");
}

File: competitive_programming/codeforces/11_golang/112a_petya_and_strings.go
package main

import (
	"fmt"
	"strings"
)

func petyaAndStrings(a, b string) int {
	left, right := strings.ToLower(a), strings.ToLower(b)
	if left < right {
		return -1
	}
	if left > right {
		return 1
	}
	return 0
}

func main() {
	if petyaAndStrings("aaaa", "aaaA") != 0 || petyaAndStrings("abs", "Abz") != -1 || petyaAndStrings("abcdefg", "AbCdEfF") != 1 {
		panic("petya failed")
	}
	fmt.Println("112A petya and strings ok")
}
